File: Back/NodeJS/modules/bets/src/application/features/answer-bet/BetAnsweredNotification.ts
import { INotification } from "../../../../../shared/application/Request/INotification";

export class BetAnsweredNotification extends INotification {
    Name: string = BetAnsweredNotification.name;

    constructor(public BetId: string, public MemberId: string, public Answer: boolean){
        super();
    }

}

File: Back/.Net/Modules/Bets/BetFriends.Bets.Application/Features/AnswerBet/BetAnsweredNotification.cs
using BetFriends.Bets.Domain.Bets;
using BetFriends.Bets.Domain.Members;
using MediatR;

namespace BetFriends.Bets.Application.Features.AnswerBet;

public record BetAnsweredNotification(BetId BetId, MemberId MemberId, bool Answer) : INotification;
